src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="ma-keywords-generator">
-    <div class="card">
-      <div class="container">
-        <div class="input-container">
-          <div class="ma-header">
+  <div class="ma-keywords-generator max-w-6xl mx-auto p-8">
```

Comment:
If we are not using the `ma-keywords-generator` class, we can remove it. Or if we want to improve readability, we can define it in `style` blocks as in the example.

```suggestion
  <div class="ma-keywords-generator">
```

```scss
.ma-keywords-generator {
  @apply max-w-6xl mx-auto p-8;
}
```